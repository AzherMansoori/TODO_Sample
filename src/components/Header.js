import React from 'react';
import {Text, View, StyleSheet} from 'react-native';

export default function Header(props) {
  return (
    <View style={styles.container}>
      <Text style={styles.header}>{props.title}</Text>
      <Text style={styles.header}>{props.title}</Text>
    </View>
  );
}

const styles = StyleSheet.create({
  header: {
    fontSize: 25,
    color: 'darkblue',
    marginLeft: 5,
    fontWeight: 'bold',
  },
  container: {
    width: '100%',
    height: 50,
    justifyContent: 'center',
  },
});
