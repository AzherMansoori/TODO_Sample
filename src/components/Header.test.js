import React from 'react';
import {shallow} from 'enzyme';
import Header from './Header';
import renderer from 'react-test-renderer';

describe('Header', () => {
  describe('Rendering', () => {
    it('renders correctly', () => {
      const tree = renderer.create(<Header />);
      expect(tree).toMatchSnapshot();
    });
    it('should match to snapshot', () => {
      const component = shallow(<Header title="test title" />);
      expect(component).toMatchSnapshot();
    });
  });
});
