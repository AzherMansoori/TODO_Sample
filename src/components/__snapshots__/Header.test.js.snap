// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Rendering renders correctly 1`] = `
<View
  style={
    Object {
      "height": 50,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Text
    style={
      Object {
        "color": "darkblue",
        "fontSize": 25,
        "fontWeight": "bold",
        "marginLeft": 5,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "darkblue",
        "fontSize": 25,
        "fontWeight": "bold",
        "marginLeft": 5,
      }
    }
  />
</View>
`;

exports[`Header Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
